forum/users/serializers.py

```diff
@@ -56,5 +61,19 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
         if roles_data:
             user.roles.set(roles_data)
+            
+        token = RefreshToken.for_user(user).access_token
+        
+        
+        activation_url = f"http://localhost:8000{reverse('activate', kwargs={'token': str(token)})}"
```

Comment:
You're hardcoding the activation URL to `http://localhost:8000`. This will cause issues when deploying to production, where the actual URL will be different.
Make the domain configurable through environment variables, so you can set it to the correct value in production.
```
activation_url = f"{settings.FRONTEND_URL}{reverse('activate', kwargs={'token': str(token)})}"
```
In your `settings.py`:
```
FRONTEND_URL = os.getenv('FRONTEND_URL', 'http://localhost:8000')
```